suro-elasticsearch/src/main/java/com/netflix/suro/sink/elasticsearch/ElasticSearchSink.java

```diff
@@ -259,23 +275,29 @@ public void run() {
                 HttpResponse response = null;
                 try {
                     response = client.executeWithLoadBalancer(request.first());
+                    stopwatch.stop();
```

Comment:
if execute throws an exception, then we won't stop the watch.  should we wrap above execute in a small try-finally.